docs/admin/install.md

```diff
@@ -4,22 +4,22 @@ Here you will find detailed instructions on how to **install** and **configure**
 
 ## Prerequisites
 
-- The plugin is compatible with Nautobot 1.6.0 and higher.
+- The app is compatible with Nautobot 2.0.6 and higher.
 - Databases supported: PostgreSQL, MySQL
 
 !!! note
     Please check the [dedicated page](compatibility_matrix.md) for a full compatibility matrix and the deprecation policy.
 
 ### Access Requirements
 
-The plugin does not directly communicate with NetBox, data is imported by first running a Django data dump command from the CLI of the NetBox server then providing the JSON export to a management command on the Nautobot server. Detailed instructions are located on the [Using the App](../user/app_use_cases.md) page.
+The app does not directly communicate with NetBox, data is imported by first running a Django data dump command from the CLI of the NetBox server then providing the JSON export to a management command on the Nautobot server. Detailed instructions are located on the [Using the App](../user/app_use_cases.md) page.
 
 ## Install Guide
 
 !!! note
-    Plugins can be installed manually or using Python's `pip`. See the [Nautobot documentation](https://nautobot.readthedocs.io/en/latest/plugins/#install-the-package) for more details. The pip package name for this plugin is [`nautobot-netbox-importer`](https://pypi.org/project/nautobot-netbox-importer/).
+    Apps can be installed manually or using Python's `pip`. See the [nautobot documentation](https://nautobot.readthedocs.io/en/latest/plugins/#install-the-package) for more details. The pip package name for this app is [`nautobot-netbox-importer`](https://pypi.org/project/nautobot-netbox-importer/).
```

Comment:
Out of curiosity - how does one "manually" install an app?

Reply:
Will be fixed in the next re-bake.

Reply:
> Out of curiosity - how does one "manually" install an app?

Naturally, you download the package, unzip it, then copy over file by file into site-packages 🤓